pyproject.toml

```diff
@@ -1,16 +1,16 @@
 [tool.poetry]
 name = "PyPiCount"
 version = "0.1.0"
-description = "A counting program used for different Pythong packages and modules."
+description = "A counting program used for different Python packages and modules."
 authors = ["Adriana Solis <solisa@allegheny.edu>, Alexis Caldwell <caldwella2@allegheny.edu>, Rachael Harris <harrisr@allegheny.edu>, Ramon Guzman <guzmanr@allegheny.edu>, Ryan Hilty <hiltyr@allegheny.edu>"]
 license = "MIT"
 packages = [
-    { include = "py_counter.py", from="src" },
-    { include = "src/py_counter.py"},
+    { include = "py_counter.py", from="pypi_count" },
+    { include = "pypi_count/py_counter.py"},
```

Comment:
Individual files should not be included; this field should specify the *python package*, which in your case would be `pypi_count`. Additionally, the same content should not be included twice.

Reply:
@solisa986